### ISSUE TITLE
Multi-Class MOT

### ISSUE THREAD

**Issue by ed-muthiah** [timmeinhardt/trackformer#10]
Hello, 

Does TrackFormer support multi-class multi-object tracking? If so how do I go about training this?

Thanks :)

**Comment by timmeinhardt** [comment]
The MOT and CrowdHuman dataset only support a single class. But the extension to multiple classes should be very straight forward and require little implementation effort.

**Comment by JackWoo0831** [comment]
Hello,
Now I want to try multi-class MOT, but I  met the assert error:
`in util/box_ops.py:
assert (boxes1[:, 2:] >= boxes1[:, :2]).all()`

When I trained the single class, it happened once only because a large learning rate. but now the learning rate is proper and it doesn't work when I decrease the lr.

Have you met this issue?

Thanks.

**Comment by timmeinhardt** [comment]
I never tried to convert the repo to multiple classes. This is definitely possible and should be straightforward. But you need to change the code accordingly.

**Comment by timmeinhardt** [comment]
Great! Is it working now?

**Comment by JackWoo0831** [comment]
I'm still training, and it seems good. Thank u very much for your excellent work.
After some epochs I plan to add some code of showing the class name in visdom, hahaha.

**Comment by timmeinhardt** [comment]
Class name in visdom sounds good. :)

**Comment by bilalorhanlr** [comment]
> Merhaba, Şimdi bu sorunu çözdüm. Çoklu sınıfta eğitim almak için sadece birkaç kodu değiştirmeniz yeterlidir.
> 
> 1. Modeller/init.py içindeki 'num_classes' değerini değiştirin. örneğin, VisDrone veri kümesindeki 4 sınıfın nesnelerini izlemek istiyorum:
> 
> ```python
> elif args. dataset==' VisDrone':# Edited
>     # num_classes=1
>     num_classes=4
> ```
> 
> daha sonra sınıflandırma kafası buna göre FC katmanındaki boyutu değiştirecektir. 2.En önemlisi, COCO biçimi ek açıklamaları oluştururken, kategori kimliği 1 ile başlamalı ve birer birer artmalıdır, örneğin:
> 
> ```python
> annotations['categories']=[{"supercategory":"vehicle","name":"car","id":1},
> {"supercategory":"vehicle","name":"van","id":2},
> {"supercategory":"vehicle","name":"truck","id":3},
> {"supercategory":"vehicle","name":"bus","id":4}]
> ```

Hello there, is it work ?
